### PR TITLE
grc: Improved shell integration and caveats

### PR DESCRIPTION
**Pull request by KarlAmort**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

- `arc.zsh` was included in the current release and no longer needs the `if build.head?` qualification. The script for fish shell should also be copied to `#{etc}`.

- Expanded the caveats to address all three shells.

### CODE REVIEW

File: Formula/grc.rb [hunk 1]
```diff
@@ -34,8 +35,17 @@ def post_install
   end
 
   def caveats; <<-EOS.undent
-    New shell sessions will start using GRC after you add this to your profile:
-      . #{etc}/grc.bashrc
+    To use GRC automatically for new shell sessions:
+
+      Bash: add to ~/.profile:
+        . /usr/local/etc/grc.bashrc
```
Comment (bfontaine) [inline Formula/grc.rb]: Use `#{etc}` instead of `/usr/local/etc`. Not all users have Homebrew under `/usr/local`.

File: Formula/grc.rb [hunk 2]
```diff
@@ -34,8 +35,17 @@ def post_install
...
+
+      ZSH: add to ~/.zshrc
+        [[ -s "/etc/grc.zsh" ]] && source /etc/grc.zsh
```
Comment (bfontaine) [inline Formula/grc.rb]: Use `#{etc}` here and below.

**Review by bfontaine — CHANGES_REQUESTED** [review]

File: Formula/grc.rb [hunk 3]
```diff
@@ -34,8 +35,17 @@ def post_install
   end
 
   def caveats; <<-EOS.undent
-    New shell sessions will start using GRC after you add this to your profile:
-      . #{etc}/grc.bashrc
+    To use GRC automatically for new shell sessions:
+
```
Comment (MikeMcQuaid) [inline Formula/grc.rb]: Remove this empty line.

File: Formula/grc.rb [hunk 4]
```diff
@@ -34,8 +35,17 @@ def post_install
...
+      Bash: add to ~/.profile:
+        . /usr/local/etc/grc.bashrc
+
+      ZSH: add to ~/.zshrc
```
Comment (MikeMcQuaid) [inline Formula/grc.rb]: I'm not convinced we need to add these per-shell. You could instead say:
```
New shells will use GRC if you add the relevant file to your profile e.g.:
  . #{etc}/grc.bashrc
```

**Pull request closed without merging**